recipes/deployer.rb

```diff
@@ -0,0 +1,53 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# Recipe:: deployer
+#
+# Configures the Deployer in a Search Head Cluster
+
+fail 'Deployer installation not currently supported on windows' if platform_family?('windows')
+
+fail 'SH clustering is disabled. Set node["splunk"]["config"]["sh_cluster"] to true' unless node['splunk']['config']['sh_cluster']
+
+include_recipe 'cerner_splunk::_install_server'
+
+password_file = File.join node['splunk']['external_config_directory'], 'password'
+search_heads = CernerSplunk.my_cluster_data(node)['sh_cluster']
+
+fail 'Search Heads are not configured for sh clustering in the cluster databag' if search_heads.nil? || search_heads.empty?
+
+execute 'apply-shcluster-bundle' do
+  command "cat #{password_file} | xargs -I{} #{node['splunk']['cmd']} apply shcluster-bundle -target '#{search_heads.first}' --answer-yes -auth admin:{}"
+  environment 'HOME' => node['splunk']['home']
+  action :nothing
+end
+
+attributes = node['splunk']['apps']
+
+attributes_bag = CernerSplunk::DataBag.load(attributes['bag']) || {}
+
+cluster_data = CernerSplunk.my_cluster_data(node) || {}
+
+cluster_bag = CernerSplunk::DataBag.load(cluster_data['apps'], pick_context: ['deployer-apps']) || {}
+
+bag_bag = CernerSplunk::DataBag.load(cluster_bag['bag']) || {}
```

Comment:
That was copy/pasta from _configure_apps recipe. I renamed it to global_apps_bag because that is what it is used here for.
995d3896515395d882c62fc95a33b659518485bf